justfile

```diff
@@ -183,16 +183,37 @@ wipe-app:
     #!/usr/bin/env bash
     set -euxo pipefail
     echo "Wiping native 10101 app"
+
+    # Locate macOS 10101 application directory if it exists
+    if [ -d "$HOME/Library/Containers/" ]; then
+        FOUND_DIR=""
+        for dir in $HOME/Library/Containers/*/Data/Library/Application\ Support/finance.get10101.app; do
+            if [ -d "$dir" ]; then
+                FOUND_DIR="$dir"
+                break  # Exit loop after the first match. Remove if you want all matches.
+            fi
+        done
+        # default to dummy dir if nothing got found to ensure nothing unnecessary gets deleted
+        MACOS_PATH="${FOUND_DIR:-/path/to/dummy/directory}"
+    fi
+
+    # If no path was found, use a dummy path to avoid errors
+    if [[ ! $MACOS_PATH || ! ${MACOS_PATH//[[:space:]]/} ]]; then
```

Comment:
Changing this to
```
    # If no path was found, use a dummy path to avoid errors
    if [[ -z ${MACOS_PATH+x} || ! $MACOS_PATH || ! ${MACOS_PATH//[[:space:]]/} ]]; then
```
Seems to work

Reply:
Nice find! Can you open a PR for this?

Reply:
#1465